translations/VoltFish-es.ts

```diff
@@ -45,27 +45,27 @@
     </message>
     <message id="about-text">
         <source></source>
-        <translation>Un cliente no oficial para Sailfish OS de &lt;a href=&quot;https://moosh.im/mooshimeter/&quot;&gt;Mooshimeter.&lt;/a&gt; Multímetro de bajo consumo de energía Bluetooth</translation>
+        <translation>Un cliente no oficial de &lt;a href=&quot;https://moosh.im/mooshimeter/&quot;&gt;Mooshimeter.&lt;/a&gt; para Sailfish OS. Un multímetro Bluetooth de bajo consumo de energía</translation>
     </message>
     <message id="battery-abbrev">
         <source></source>
         <translation>Bat</translation>
     </message>
     <message id="sampling-rate">
         <source></source>
-        <translation>Frecuencia de muestreo</translation>
+        <translation>Frecuencia muestreo</translation>
     </message>
     <message id="buffer-depth">
         <source></source>
-        <translation>Profundidad de búfer</translation>
+        <translation>Profundidad búfer</translation>
     </message>
     <message id="settings">
         <source></source>
         <translation>Ajustes</translation>
     </message>
     <message id="firmware-note">
         <source></source>
-        <translation type="unfinished"></translation>
+        <translation>nota del programaLógico</translation>
```

Comment:
I don't speak any spanish, but this does not seem to be the intended message.
I moved to ID based translations, which solved a couple of problems. "firmware-note" just a placeholder, the english text to be translated in this case is:
> Note:
> requires Mooshimeter firmware 147xxx

see: https://github.com/sigurg/VoltFish/blob/d3f6b6ee3a43c1d8efb0c6e4da2cc9fef4ed33fe/translations/VoltFish.ts#L64